cmd/flowrunner/testdata/flows/all_actions.json

```diff
@@ -39,8 +39,10 @@
                         {
                             "uuid": "7bd8b3bf-0a3c-4928-bc46-df416e77ddf4",
                             "type": "save_contact_field",
-                            "field_uuid": "f06c5b73-eb0d-4417-b7e0-4f650ed30dc8",
-                            "field_name": "Activation Token",
+                            "field": {
```

Comment:
This format is more like how we reference groups/labels/contacts in flows. Key won't be used, and might be wrong, but like group name etc, makes flow more readable.